website/project/model.py

```diff
@@ -2756,7 +2755,7 @@ def remove_contributor(self, contributor, auth, log=True):
         self.save()
 
         #send signal to remove this user from project subscriptions
-        auth_signals.contributor_removed.send(contributor, node=self)
+        project_signals.contributor_removed.send(contributor, node=self)
```

Comment:
Since contributor_removed is a project-related signal, it'd make more sense to have the project/node be the sender. Go ahead and make the change if it's trivial; otherwise nbd.